packages/content-management/user_profiles/src/components/user_missing_tip.tsx

```diff
@@ -11,25 +11,47 @@ import { FormattedMessage } from '@kbn/i18n-react';
 import { EuiIconTip, IconType } from '@elastic/eui';
 import React from 'react';
 
-export const NoCreatorTip = (props: { iconType?: IconType }) => (
+export const NoCreatorTip = (props: { iconType?: IconType; includeVersionTip?: boolean }) => (
   <NoUsersTip
     content={
-      <FormattedMessage
-        id="contentManagement.userProfiles.noCreatorTip"
-        defaultMessage="Creators are assigned when objects are created"
-      />
+      <>
+        <FormattedMessage
+          id="contentManagement.userProfiles.noCreatorTip"
+          defaultMessage="Creators are assigned when objects are created "
+        />
+        {props.includeVersionTip && (
+          <>
+            {' '}
+            <FormattedMessage
+              id="contentManagement.userProfiles.noCreatorTipVersion"
+              defaultMessage=" (after version {version})"
+              values={{ version: '8.14' }}
+            />
+          </>
+        )}
+      </>
```

Comment:
nit: I had a think about this, what if we replaced the no creator message we have with something of this sort; 

```tsx
<FormattedMessage
  defaultMessage="Item was created in a release {version} that didn't persist the creator attribute"
  values={{ version: includeVersionTip ? '< 8.14' :  '' }}
/>
```

I think this explains why the creator attribute value might be missing for the current record, as opposed to  explaining why other fields have a creator attribute.

Reply:
I like that suggestion @eokoneyo ! 👍

Reply:
> ... both the original text and the suggested text have a problem I worry about: there are more edge cases when a creator might be missing, for example, a dashboard created by an anonymous user or via API. Both texts don't mention these edge cases, but I think, @eokoneyo, the one you suggested ignores these possibilities a bit more than the original one

Hmmm... I see what you mean about the situation where we are unable to determine the user, sounds like something we could get some guidance from @florent-leborgne for